src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -873,227 +873,5 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
             positionInfo.Free();
             return (builderAppendCallsArray.ToImmutableAndFree(), builderPatternExpectsBool ?? false, positionInfoArray.ToImmutableAndFree(), baseStringLength, numFormatHoles);
         }
-
-        private BoundExpression BindInterpolatedStringHandlerInMemberCall(
```

Comment:
>BindInterpolatedStringHandlerInMemberCall [](http://example.com/codeflow?start=32&length=41)

Moved as a local function `bindInterpolatedStringHandlerInMemberCall` to `Binder_Expressions.cs`

Reply:
I would have a slight leaning towards keeping this where it is. I won't block on it, but I did put this here intentionally, rather than making it a local function in CoerceArguments, because I wanted to keep all the interpolation logic together.

Reply:
> I would have a slight leaning towards keeping this where it is. I won't block on it, but I did put this here intentionally, rather than making it a local function in CoerceArguments, because I wanted to keep all the interpolation logic together.

The primary reason for the move is to constrain access to ```MemberAnalysisResult,.ArgsToParamsOpt``` to as few helpers that anyone can call in the future as possible. I was able make other helper(s) local functions, I wouldn't otherwise. I find that more important than a desire to keep interpolation logic in a separate file.